docs/api-references/docs.md

```diff
@@ -421,6 +421,19 @@ string
 <p>PriorityClassName of Backup Job Pods</p>
 </td>
 </tr>
+<tr>
+<td>
+<code>backoffRetryPolicy</code></br>
+<em>
+<a href="#backoffretrypolicy">
+BackoffRetryPolicy
```

Comment:
Can you add doc for how these three parameters work together to support restart policy?

Reply:
ok